amass/config.go

```diff
@@ -162,3 +165,21 @@ func (c *AmassConfig) GetAPIKey(source string) *APIKey {
 	}
 	return nil
 }
+
+// ExcludeDisabledServices returns a filtered list of input services excluding DisabledServices
+func (c *AmassConfig) ExcludeDisabledServices(services []AmassService) []AmassService {
+	enabled := make([]AmassService, 0)
+	for _, service := range services {
+		include := true
+		for _, match := range c.DisabledServices {
+			if match == service.String() {
+				include = false
+				break
+			}
+		}
+		if include {
+			enabled = append(enabled, service)
+		}
+	}
+	return enabled
+}
```

Comment:
Please add a test for this asserting both valid and invalid input.

Reply:
Done.